packages/web-app-user-management/src/views/Groups.vue

```diff
@@ -1,5 +1,64 @@
 <template>
   <div>
+    <app-template
+      ref="template"
+      :loading="loadResourcesTask.isRunning"
```

Comment:
```suggestion
      :loading="loadResourcesTask.isRunning || !loadResourcesTask.last"
```
`isRunning` doesn't evaluate to `true` if the task was never started (yet).